domain-models-runtime/src/test/resources/log4j2-test.properties

```diff
@@ -11,7 +12,7 @@ appenders = console
 appender.console.type = Console
 appender.console.name = STDOUT
 appender.console.layout.type = PatternLayout
-appender.console.layout.pattern = %d{dd MMM yyyy HH:mm:ss:SSS} %-5p %C{1} [%X{reqId}] %m%n
+appender.console.layout.pattern = %d{HH:mm:ss} [$${FolioLoggingContext:requestid}] [$${FolioLoggingContext:tenantid}] [$${FolioLoggingContext:userid}] [$${FolioLoggingContext:moduleid}] %-5p %-20.20C{1} %m%n
```

Comment:
Can you add a note to the https://github.com/folio-org/raml-module-builder/blob/master/doc/upgrading.md that RMB based modules should copy https://github.com/folio-org/raml-module-builder/blob/master/domain-models-runtime/src/test/resources/log4j2-test.properties into their /src/test/resources directory?